# Name of the app to launch
string name 
---
# true if app started, false otherwise
bool started
# if app did not start, error code for classifying start failure.  See
# StatusCodes.msg for common codes.
int32 error_code
# response message for debugging
string message
# Namespace where the app interface can be found
string namespace  

